pkgs/development/python-modules/dufte/default.nix

```diff
@@ -34,6 +34,10 @@ buildPythonPackage rec {
   '';
   checkInputs = [ pytestCheckHook ];
   pythonImportsCheck = [ "dufte" ];
+  pytestFlagsArray = [
+    # we don't have the "exdown" package (yet)
```

Comment:
I am going to do a PR which adds exdown.

Reply:
Even so, I think this test just renders the readme and isn't useful enough to warrant the extra build-time dependency.

Reply:
Already did it so whatever.